CfgLoadouts.hpp

```diff
@@ -95,10 +108,9 @@ class CfgLoadouts {
     #include "Loadouts\csat_sama_hex.hpp"
   };
 
-  class potato_msv { // MSV
+  class potato_msv { // Opfor MSV
     #include "Loadouts\msv_csat_sama_hex.hpp"
   };
-
```

Comment:
This space should still be here I think, make sure it matches the formatting on the other sections